WordPress/src/main/res/layout/stats_list_cell.xml

```diff
@@ -27,7 +27,8 @@
             android:layout_marginEnd="10dp"
             android:visibility="gone"
             android:contentDescription="@string/open_external_link_desc"
-            android:src="@drawable/ic_external_blue_wordpress_18dp" />
+            android:src="@drawable/ic_external_blue_wordpress_18dp"
+            android:tint="@color/blue_wordpress" />
```

Comment:
Can you elaborate on the pattern here @theck13 ? It seems that you keep the "colored" drawable (even though there's now the "white" version of it available) but still set its tint (to the color already in the resource).